language/en/interface.json

```diff
@@ -1665,6 +1667,10 @@
 				"title": "%{role1}/%{role2}",
 				"description": "%{role1}\n\nand/or\n\n%{role2}"
 			}
+		},
+		"quicksharetotarget": {
```

Comment:
```suggestion
		"quickShareToTarget": {
```

Reply:
I don't know how to handle it in terms of action name. If action name is snakeCase then the keybind doesn't work

Reply:
The entry in the `orderMenu` subtable must be a verbatim match of the CMD name, but the `quicksharetotarget` subtable can have any arbitrary name, and this should be pascalCase'd